packages/jaeger-ui/src/components/DeepDependencies/Header/index.tsx

```diff
@@ -71,6 +81,15 @@ export default class Header extends React.PureComponent<TProps, TTempState> {
             />
           )}
         </div>
+        <div className="DdgHeader--attrHeader">
```

Comment:
The subbheader is not really for informational display, so maybe something indicating configuration of the view is more accurate? E.g. `controlsHeader`?